### PR TITLE
frei0r-plugins: update to 1.7.0.

### PR DESCRIPTION
**Pull request by tibequadorian**
Also added build options for cairo and gavl.

### CODE REVIEW

File: srcpkgs/frei0r-plugins/template [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 maintainer="Diogo Leal <diogo@diogoleal.com>"
-license="GPL-2"
+license="GPL-2.0-or-later"
 homepage="http://frei0r.dyne.org/"
 distfiles="https://files.dyne.org/frei0r/releases/frei0r-plugins-${version}.tar.gz"
-checksum=e0c24630961195d9bd65aa8d43732469e8248e8918faa942cfb881769d11515e
+checksum=1b1ff8f0f9bc23eed724e94e9a7c1d8f0244bfe33424bb4fe68e6460c088523a
+
+build_options="cairo gavl"
+build_options_default="cairo"
```
Comment (ericonr) [inline srcpkgs/frei0r-plugins/template:16]: Why not enable both?

Also, their build system seems to search for OpenCV as well. Do we want to enable that?

Comment (tibequadorian) [reply srcpkgs/frei0r-plugins/template:16]: I thought to keep dependencies by default rather minimal and cairo is probably installed anyway, gavl is not.
I tried OpenCV (3 and 4) as well but it failed with different error messages. Maybe someone else knows how to do this...

Comment (ericonr) [reply srcpkgs/frei0r-plugins/template:16]: Fair enough, this should be adding more functionality as is already.

Can you leave a comment about OpenCV also being a possible dependency? `# XXX: ...`

**Comment by ericonr** [comment]
Ah, which software have you tried it with?

**Comment by tibequadorian** [comment]
> Ah, which software have you tried it with?

kdenlive, it worked fine

File: srcpkgs/frei0r-plugins/template (hunk 1 above)
Comment (tibequadorian) [reply srcpkgs/frei0r-plugins/template:16]: sry what do you mean by that `# XXX: ...` what comes instead of `XXX` ? :smile:

Comment (tibequadorian) [reply srcpkgs/frei0r-plugins/template:16]: ohh I didn't know it was a convention! nvm

**Pull request merged**